### PR TITLE
Added styles to some connectors

### PR DESCRIPTION
**Pull request by tamersaadeh**
I created a node module that deals with generating the HTML page that is going to be sent to the user of Locker.
# Implemented common styles to the following connectors:

```
1. Amazon History
2. Chrome History
3. Firefox History
4. Instagram
5. LinkedIn
6. Twitter
7. Foursquare
8. Facebook
```

These are all of the connectors that have clients in javascript. I will not be able to implement something similar to the other, since they are written in Python and I have no knowledge of it.
# Note:

All of the colors are the same in all clients this can be easily changed/modified by changing two elements of the array parameter of formatHTML(). First element is the color of the background of the header, and the second element is the color of the text in the header.

### CODE REVIEW

**Comment by tamersaadeh** [comment]
Fixed minor issues with the styles that will cause the elements of the page to be in weird places with different screen sizes and when the content of the page is longer than the screen (ie requires scrolling). Also, implemented it in IMAP's demo.

**Comment by tamersaadeh** [comment]
If this gets pulled, I will be able to contribute a wiki for the uses of this new module.

File: Common/node/html.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
...SanG0osKcC3gWuBjwAexEtzAmAu8hKgYtIHJCkf9qixXfJ828SR5YOo3RJJ2QwY8hyi73e/qnMX3yonJz+cAzyBK0GoAHQa8E7gG2EL5F2P3119HbCSzIvWbIEl7YBh4MfA/lHsPHd/Bn1cDXwBeRvS4asCNAscCv0fMMF9NdH13l2Td09Z7d4u8GM//Q2IXIkmquxYx4/3zxAz44t65pwG//b2zTexaeTcxTPkiYGnqk0+piusFq2CImK35ZOJp83jiQtkXmNf1301367/tt1GF2Hv3euBvge9jARlJzbIMOAN4M3AkUfcdJveqmInue+ca4F5iGfJ5xKqgW2hY1bfdYaDv2gjRel4BPInYX/0xwMHENqizp/l1NhFlEm8nWuUXE/vwNmZjAEnaTkbcO3+JGNM+DDiU2FZ1ujYSO7qtJFYBXU6sArqFaKGrw0CfuTlEwB9IXJh7E+NGU5XRbQNjRPf9rcRFuYoYp5ekQbEEOITY1noforG0s/Ljxb3zQSK878R7504Z6L1R1IffkbpsJShJ/bSz+yZ475QkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIk7Y4sz/P4TZalPhZJkjRDRY63gKHUByNJkvbIcAs4hgh2SZJUP5HleZ6PFM11SZJUL3mek+f5iGPokiTVmI1ySZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSZIkSdLA+/8TSrrZiDidFgAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxMS0wNC0yMFQyMjozNDowOCswMDowMHLRX9AAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTEtMDQtMjBUMjI6MzQ6MDgrMDA6MDADjOdsAAAAF3RFWHRwbmc6Yml0LWRlcHRoLXdyaXR0ZW4ACKfELPIAAAAASUVORK5CYII=' height='30px' width='30px'></a></div>"
```
Comment (tamersaadeh) [inline Common/node/html.js:35]: I know that there is a link that does the exact thing as the logo, but its more to stay with conventional look/feel for the pages.

**Comment by tamersaadeh** [comment]
I have noticed that the code I have updated have changed significantly to disallow this pull to be merged easily. Should I merge the current dev into my code and update it?

**Comment by smurthas** [comment]
Hey tamersaadeh, thanks for the help. We are in the middle of doing a big Connector refactor, could you refactor and resubmit this for just the Facebook, Twitter and Foursquare Connectors as they look right now (broke out auth, api, and sync code into separate files - checkout Twitter for a highly commented example and https://github.com/quartzjer/Locker/wiki/The-Anatomy-of-a-Connector for more info on the break out).

Thanks!

**Comment by tamersaadeh** [comment]
Ok, great and I'm always ready to help! I will try to update it, but I think I might not be able to until the May 17th. I will just keep this pull open and update it then.

Thanks for the update!

**Comment by tamersaadeh** [comment]
This update was not as easy as I thought it would be. I'm closing this pull and will create a new one when I do all of the things you mentioned above. :)

**Pull request closed without merging**